amgut/templates/sitebase.html

```diff
@@ -1,8 +1,8 @@
 {% from future.utils import viewitems %}
 {% from amgut import media_locale %}
 {% from amgut.connections import ag_data, redis %}
-{% set human_participants, animal_participants, environmental_samples, kit_verified = ag_data.get_menu_items(skid) %}
-{% set base_user_id = ag_data.get_user_for_kit(skid) %}
+{% set human_participants, animal_participants, environmental_samples, kit_verified = ag_data.get_menu_items(current_user) %}
+{% set base_user_id = ag_data.get_user_for_kit(current_user) %}
```

Comment:
Changes to sitebase will have broad implications. Any impact across the templates?